lib/views/custom_widgets/photo_collage.dart

```diff
@@ -351,16 +354,22 @@ class PhotoCollageState extends State<PhotoCollage> with UiLoggy {
   }
 
   Future<Uint8List?> getCollageImage({required double pixelRatio, ExportFormat format = ExportFormat.jpgFormat, int jpgQuality = 80}) async {
-    const delay = Duration(milliseconds: 20);
     if (format == ExportFormat.pngFormat) {
-      return screenshotController.capture(pixelRatio: pixelRatio, delay: delay);
+      return screenshotController.capture(pixelRatio: pixelRatio);
     }
-    final image = await screenshotController.captureAsUiImage(pixelRatio: pixelRatio, delay: delay);
-    // Lib by default uses ui.ImageByteFormat.png for capture, encoding is what takes long.
+
+    // Await frame render, should workaround the black image issue
+    await waitForPostFrameCallback();
```

Comment:
Great if this works!

Reply:
Should be above the PNG capture though!

Reply:
Oh, this is what happens when hacking at times one should be sleeping...